### PR TITLE
Add RadioPublic overlay

### PR DESCRIPTION
**Pull request by chrisrhoden**
Bring back the download button with the option to get an MP3 or the RP app
Display the overlay when the content is paused after it has started playing.

### CODE REVIEW

**Comment by chrisrhoden** [comment]
So, the simple answer is that you can dismiss by resuming when you induced the overlay by pausing. There are a couple of other circumstances that you might trigger the overlay in, so I recognize the need.

I'll make hitting escape do what we want, though I do want to maintain that as a responsibility of the overlay content rather than a global default behavior of the overlay wrapper, again to allow the (perhaps unlikely but still better to leave our options open) possibility of takeovers that we actually *don't* want users to bypass.

**Comment by cavis** [comment]
Could be just an edge case - I was hitting the download button before even playing the content.  So I was left with a scenario where I couldn't get out of the overlay without _starting_ playback.

Also seem to recall the RP content not being horizontally centered, which can look a bit funny on very-wide players.  (Again, probably an edge case).

Also also - this doesn't count as my review!

**Comment by chrisrhoden** [comment]
@cavis consider this my solicitation of your review then. I added escape to dismiss the thing

File: src/app/embed/embed.component.ts [hunk 1]
```diff
@@ -19,9 +19,19 @@ const PYM_CHILD_ID_PARAM = 'childId';
...
-        <p>Placeholder for overlay content (could put share modal here if we wanted)</p>
+        <div class="app-overlay" (window:keydown)="handleKeypress($event)">
+          <p>Never miss an episode from <strong>{{this.subtitle}}</strong> and other great podcasts when you download the free RadioPublic app.</p>
+          <ul class="app-selection">
+            <li *ngIf="!isiOSDevice"><a [href]="playStoreLink()"><img src="/assets/images/google-play.png" alt="Google Play Store" /></a></li>
+            <li *ngIf="!isAndroidDevice"><a [href]="appStoreLink()"><img src="/assets/images/app-store.svg" alt="App Store" /></a></li>
+            <li *ngIf="isiOSDevice">or the <a [href]="playStoreLink()">Google Play Store</a></li>
+            <li *ngIf="isAndroidDevice">or the <a href="appStoreLink()">App Store</a></li>
+          </ul>
+          <p class="aside" *ngIf="downloadRequested">You can also <a [href]="audioUrl" [target]="subscribeTarget">download the audio file</a> if you're on a computer.</p>
```
Comment (cavis) [inline src/app/embed/embed.component.ts]: What's the reason for the `[target]="subscribeTarget"`?

Comment (chrisrhoden) [reply src/app/embed/embed.component.ts]: so it opens in a new tab if you're into that sort of thing

Comment (chrisrhoden) [reply src/app/embed/embed.component.ts]: (we're staunchly against it here at RP, but I hear that PRX prefers to open links in a new window)

**Comment by chrisrhoden** [comment]
@cavis I think this addresses your concerns. LMK if we're still blocked here, I'd love to see this get merged today if possible!

**Pull request merged**